### PR TITLE
[WWST-4544, WWST-4545] add fingerprints for eWeLink and SONOFF smart plug 

### PR DESCRIPTION
**Pull request by ZouLianghuiS**
pull request for WWST-4544

### CODE REVIEW

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -33,6 +33,8 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B05", outClusters: "0003, 0006, 0019", manufacturer: "Centralite Systems", model: "4200-C", deviceJoinName: "Centralite Smart Outlet"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FC7C", outClusters: "0005, 0019, 0020", manufacturer:"IKEA of Sweden", model: "TRADFRI control outlet", deviceJoinName: "IKEA TRÅDFRI control outlet"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0402, 0B05, FC01, FC02", outClusters: "0003, 0019", manufacturer: "iMagic by GreatStar", model: "1113-S", deviceJoinName: "Iris Smart Plug"
+                fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "SmartPlug(SA-003)"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Can you double check the join name? It doesn't match what's on the ticket.

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 2]
```diff
@@ -33,6 +33,8 @@ metadata {
...
+                fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "SONOFF", model: "BASICZBR3", deviceJoinName: "Basic(R3 Zigbee)"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: This doesn't appear to be part of the WWST-4544 ticket. Is there another ticket for this? The join name isn't very clear what the device is.

Comment (ZouLianghuiS) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: coolkit requested certification for 2 devices，one is wwst-4544，another one is wwst-4545，

**Comment by tpmanley** [comment]
We requested some clarification on the join names on the tickets so will wait for the response

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 3]
```diff
@@ -33,6 +33,8 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B05", outClusters: "0003, 0006, 0019", manufacturer: "Centralite Systems", model: "4200-C", deviceJoinName: "Centralite Smart Outlet"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FC7C", outClusters: "0005, 0019, 0020", manufacturer:"IKEA of Sweden", model: "TRADFRI control outlet", deviceJoinName: "IKEA TRÅDFRI control outlet"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0402, 0B05, FC01, FC02", outClusters: "0003, 0019", manufacturer: "iMagic by GreatStar", model: "1113-S", deviceJoinName: "Iris Smart Plug"
+                fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink SmartPlug (SA-003)"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Sorry I didn't notice before but the indentation doesn't line up with the lines above.

**Comment by tpmanley** [comment]
If you need this to go right to STG can you please rebase your change on the `staging` branch?

**Review by tpmanley — APPROVED** [review]

**Pull request merged**